packages/muelu/src/Smoothers/MueLu_Amesos2Smoother_def.hpp

```diff
@@ -202,6 +196,12 @@ namespace MueLu {
       SC normalization = Nullspace->getVector(0)->norm2();
       normalization = Teuchos::ScalarTraits<Scalar>::one()/(normalization*normalization);
 
+      ArrayView<const SC> nullspace, nullspaceImp;
+      nullspaceRCP = Nullspace->getData(0);
+      nullspace = nullspaceRCP();
+      nullspaceImpRCP = NullspaceImp->getData(0);
+      nullspaceImp = nullspaceImpRCP();
+
```

Comment:
What does this change do? Looks like the code was just moved around?

Reply:
It did just move, however, it no longer has a double cache of the data, it is used on line 196 above for the norm